test/_utils.py

```diff
@@ -17,3 +20,18 @@ def approx_equal(self, other, epsilon=1e-4):
             "Size mismatch between self ({self}) and other ({other})".format(self=self.size(), other=other.size())
         )
     return torch.max((self - other).abs()) <= epsilon
+
+
+def get_cuda_max_memory_allocations() -> int:
```

Comment:
Is the typing Python 3.6 compatible? We don't yet require 3.7.

Reply:
Other than that, this looks good to me!

Reply:
yeah typing is py36+

Reply:
I think we have to be py3.5 compatible for the docs, but Travis isn't complaining...
I'll merge, but let's check that we haven't destroyed the docs in the process :)